cmake/SanitizerOptions.cmake

```diff
@@ -0,0 +1,13 @@
+if(ENABLE_SANITIZER)
```

Comment:
This should be `ENABLE_ADDRESS_SANITIZER`, per the suggestion earlier.

Reply:
Done